rukundoeric/bookstore
# Milestone 1: Project setup

## What does this do? 
*  Setup Bookstore app.
## Description of Task to be completed? 
*  In this milestone, I setup everything for bookstore.
## How should this be manually tested? 
NOTE: Make sure you have Node.js and npm or yarn installed in your PC.
- Clone this repo: `git clone https://github.com/rukundoeric/bookstore.git`
- Navigate into project directory: `cd bookstore`
- Install all packages: `yarn install or npm install`
- Start the app: `yarn test or npm test`
